browser/tsconfig.json

```diff
@@ -12,7 +12,8 @@
       "removeComments": false,
       "preserveConstEnums": false,
       "downlevelIteration": true,
-      "watch": false,
```

Comment:
I removed this because it has no effect - we would have to use https://www.typescriptlang.org/docs/handbook/configuring-watch.html#configuring-file-watching-using-a-tsconfigjson if we wanted to configure watch options. Although if this is false (the default) it's doubly-ineffective. It's not strictly related to strictNullChecks